### DISCUSSION TITLE
Respawn time not working

### DISCUSSION THREAD

**Discussion by Revenger** [Pryaxis/TShock#1871] [General]
I set restawn to 2 normal and 5 seconds  boss but the respawns are still 10 / 20 seconds.
No idea about this.

**Comment by hakusaro** [comment]
@AxeelAnder I think you might have more context on this?

**Comment by Revenger** [comment]
Respawn timers are working after a update to the latest version from what I saw today.

**Reply by hakusaro** [reply]
Can you test it out on the latest version?

**Reply by hakusaro** [reply]
(That's pre-release 8).
